numpy/_core/src/umath/matmul.c.src

```diff
@@ -596,7 +596,7 @@ NPY_NO_EXPORT void
              * Use transpose equivalence:
              * matmul(a, b, o) == matmul(b.T, a.T, o.T)
              */
-            if (o_f_blasable) {
+            if (o_transpose) {
```

Comment:
Can we be sure that this is the only change necessary or do we need to extend the test cases?

Reply:
I think I got a solution for that, see my last comment on the lines changed in `numpy/_core/tests/test_multiarray.py`.